src/debug_imgui_demo.cpp

```diff
@@ -0,0 +1,142 @@
+#include "debug_imgui_demo.h"
+
+#include <imgui/imgui.h>
+
+#include "cata_imgui.h"
+#include "ui_manager.h"
+#include "input_context.h"
+#include "translations.h"
+
+
+debug_imgui_demo_ui::debug_imgui_demo_ui() :
```

Comment:
It's preferable to keep the name `imgui_demo` over changing it to `debug_imgui_demo`.

(Just making sure the name change isn't merged, maybe you knew already.)

Reply:
No, I did not know. Naming things is hard and i'm bad at it.